snap/snapcraft.yaml

```diff
@@ -13,13 +13,17 @@ confinement: strict
 
 parts:
     novnc:
-        source: build/
+        source: https://github.com/novnc/noVNC/archive/v1.2.0.tar.gz
         plugin: dump
         stage-packages:
-            - websockify
```

Comment:
Is this a snap package? Perhaps it should be updated in that case?

Reply:
That was the Debian/Ubuntu websockify apt package, and is a bit out of date (in 18.04 at least). Not needed anymore now we're grabbing and building websockify directly in the snap